lib/module.js

```diff
@@ -205,7 +205,7 @@ Module._resolveLookupPaths = function(request, parent) {
   }
 
   var start = request.substring(0, 2);
-  if (start !== './' && start !== '..') {
+  if (request !== '.' && start !== './' && start !== '..') {
```

Comment:
The current one looks more readable to me, but feel free to PR that change if the perf difference is significant.